cibuildwheel/resources/build-platforms.toml

```diff
@@ -166,3 +166,8 @@ python_configurations = [
   { identifier = "pp39-win_amd64", version = "3.9", arch = "64", url = "https://downloads.python.org/pypy/pypy3.9-v7.3.16-win64.zip" },
   { identifier = "pp310-win_amd64", version = "3.10", arch = "64", url = "https://downloads.python.org/pypy/pypy3.10-v7.3.16-win64.zip" },
 ]
+
+[pyodide]
+python_configurations = [
+  { identifier = "cp312-pyodide_wasm32", version = "3.12.1", pyodide_version = "0.26.0a6", emscripten_version = "3.1.58" },
```

Comment:
@hoodmane Is the wheel tag `pyodide_0_26_wasm32`?

Reply:
I guess maybe it would better be `312_0` or something... If @hugovk switches CPython releases to match the calendar year then they would sort of come to us. But 312 would under calver actually be Python 2023, but since the Python release is in August and we try to update ~6 months behind we'll always be a different year.

But it's probably best not to worry too much we can switch to a different format if need be.

Reply:
I'm just trying to pick a good name for the constraints file. I think I'll use Python version, then.